### ISSUE TITLE
fix: initial loading indicator is not full screen

### ISSUE THREAD

**Issue by ravinderk** [impress-org/givewp-campaigns#201]
# Bug Report

## Developer Story
As a developer, I want to show a full-screen loading icon to indicate the user that Campaign Editor is loading.

## Related To
https://github.com/impress-org/givewp-campaigns/pull/188

## Current Behaviour
I am seeing a login indication inside the custom page content area of WordPress

## Expected Behaviour
WordPress admin must hide behind the Loading indicator and gray background.

## Bug Type
- [x] This bug describes functionality that once worked as expected in version X.X.X.
- [ ] This bug describes functionality that never worked as expected.
- [ ] I am not sure whether this functionality ever worked as expected.

## Visuals
![NZKOQAHlBC](https://user-images.githubusercontent.com/1784821/68584967-f7669980-04a6-11ea-8a4a-d38a6a954b87.gif)

## Acceptance Criteria
- [ ]  Loading indicator is full screen.

**Comment by ravinderk** [comment]
@knowler I see that you are using tailwind class in the campaign builder template but I can see loading indicator UI issue only because of tailwind CSS inject by React and browser take time to apply it.

![image](https://user-images.githubusercontent.com/1784821/68586887-b2913180-04ab-11ea-81b1-974a80aafbc1.png)

**Comment by knowler** [comment]
@ravinderk Yes, but hat is only in development. In production, the classes are extracted to a stylesheet.

**Comment by ravinderk** [comment]
@knowler You are correct. I am closing this issue. I will look into to new setup.

**Comment by ravinderk** [comment]
@knowler I am reopening this issue only for record and it will keep us reminding that we have to find a way to either use mode loader to the main app or find another way to load CSS in development mode.
